content/custom-decorators.md

```diff
@@ -112,20 +111,14 @@ Let's define a decorator that takes a property name as key, and returns the asso
 import { createParamDecorator, ExecutionContext } from '@nestjs/common';
 
 export const User = createParamDecorator(
-  (data: string, ctx: ExecutionContext) => {
-    const request = ctx.switchToHttp().getRequest();
-    const user = request.user;
-
+  (data: keyof Request['user'], { user }: Request) => {
```

Comment:
In case with Passport we actually have Request.User type (which is empty and required augmentation), but this works as well